test/src/test/java/hudson/model/queue/WideExecutionTest.java

```diff
@@ -87,6 +95,33 @@ public String getDisplayName() {
         }
     }
 
+    @Issue("JENKINS-30084")
+    @Test
+    /*
+     * this is to test that when the assigned executor is not available the flyweighttask is put into the buildable list,
+     * thus the node will be provisioned.
+     * when the flyweight task is not assigned to an offline executors the buildable list is empty.
+     *
+     */
+    public void flyWeightTaskQueue () throws IOException {
+        MatrixProject project2 = j.createMatrixProject();
+        project2.setAxes(new AxisList(
+                new Axis("axis", "a", "b")
+        ));
+        project2.scheduleBuild2(0);
+        Queue.getInstance().maintain();
+        assertTrue(Queue.getInstance().getBuildableItems().isEmpty());
```

Comment:
This all seems to be a control case which is surely covered (many times over) by existing tests.